lib/dynflow/schedulers/abstract.rb

```diff
@@ -0,0 +1,17 @@
+module Dynflow
+  module Schedulers
+    class Abstract
+
+      attr_reader :core
+
+      def ask(*args)
```

Comment:
What about making the interface more explicit about what features/methods to implement and less about the actor being used at the backend? If I implemented new interface, I would welcome to see the Abstract class and see, what methods I'm expected to implement, no matter what interface I choose.